querytest/tf_validation.go

```diff
@@ -0,0 +1,64 @@
+package querytest
+
+import (
+	"testing"
+
+	"github.com/influxdata/flux"
+	"github.com/influxdata/flux/dependencies/dependenciestest"
+	"github.com/influxdata/flux/dependencies/url"
+	"github.com/influxdata/flux/execute"
+	"github.com/influxdata/flux/execute/executetest"
+	"github.com/influxdata/flux/plan"
+	"github.com/stretchr/testify/assert"
```

Comment:
So far the Flux code base doesn't use the `github.com/stretchr/testify/assert`  testing framework. It looks like it was introduced when we added the `querytest` package and we missed it. Flux uses the https://github.com/google/go-cmp testing package. Can you refactor the querytest package to use that package instead? Then we can reduce our dependencies back down to a smaller set. Thanks